packages/fxa-content-server/app/scripts/views/post_verify/third_party_auth/set_password.js

```diff
@@ -51,6 +53,10 @@ class SetPassword extends FormView {
     }
   }
 
+  afterRender() {
+    GleanMetrics.setPasswordThirdPartyAuth.view();
```

Comment:
Just a note that we've been putting this in `logView` in other files instead of `afterRender`, but probably doesn't matter?

Reply:
I think I can answer this: I noticed as well that there is this existing pattern where we use the `recordEventMetric` case statement. I followed the same in my PR too. 🤷

Reply:
Yea I don't think this will matter 👍🏽